src/main/resources/lang/en_us.json

```diff
@@ -113,6 +113,8 @@
 	"prompt.close.save": "Save and close",
 	"prompt.close.discard": "Discard changes",
 	"prompt.close.cancel": "Cancel",
+	"prompt.open": "Open",
+	"prompt.cancel": "Cancel",
```

Comment:
Is it possible to use the cancel already there? Or rename `prompt.close.cancel` to unify it?

Reply:
This is the one that's for unifying it. I didn't touch the existing one yet because other languages might still be using that. Also pretty sure other open PRs (the multiplayer one) also add cancel buttons so I'll unify all that in a seperate PR.